### PR TITLE
Make sure all bundle changes update wallet state

### PR DESCRIPTION
**Pull request by patrickxb**
There's a bug where if you change your bundle (say change an account name) the wallet state can be stale for some time and send old data with its notifications to the frontend.

This PR fixes that so that any bundle changes update wallet state.

cc @buoyad

### CODE REVIEW

File: go/stellar/wallet_state.go [hunk 1]
```diff
@@ -166,6 +206,36 @@ func (w *WalletState) Primed() bool {
...
+	defer mctx.TraceTimed(fmt.Sprintf("WalletState.UpdateAccountEntriesWithBundle [%s]", reason), func() error { return err })()
+
+	if bundle == nil {
+		return errors.New("nil bundle")
+	}
+
+	for _, account := range bundle.Accounts {
+		a, _ := w.accountStateBuild(account.AccountID)
+		a.updateEntry(account)
+	}
```
Comment (zapu) [inline go/stellar/wallet_state.go]: should it remove accounts that don't exist anymore from the state?

File: go/stellar/wallet_state.go [hunk 2]
```diff
@@ -107,6 +107,46 @@ func (w *WalletState) BaseFee(mctx libkb.MetaContext) uint64 {
 	return w.options.BaseFee(mctx, w)
 }
 
+// AccountName returns the name for an account.
+func (w *WalletState) AccountName(accountID stellar1.AccountID) (string, error) {
```
Comment (zapu) [inline go/stellar/wallet_state.go]: so these are only used in tests right now? will they be used in other code as well?

Comment (patrickxb) [reply go/stellar/wallet_state.go]: Just in tests, but seems fine if they are needed elsewhere.  Currently they aren't...

File: go/stellar/wallet_state.go (hunk 1 above)
Comment (patrickxb) [reply go/stellar/wallet_state.go]: ok

Comment (patrickxb) [reply go/stellar/wallet_state.go]: done.

**Pull request merged**